source/air/src/focus.rs

```diff
@@ -18,6 +18,10 @@ pub fn focus_command_on_assert_id(command: &Command, assert_id: &AssertId) -> Op
             let query = Arc::new(QueryX { local: query.local.clone(), assertion });
             Some(Arc::new(CommandX::CheckValid(query)))
         }
+        CommandX::CheckSingular(..) => {
+            // TODO: what should we do here?
```

Comment:
I am not exactly sure what to do with assertion ids in Singular queries.

Reply:
This is part of the new `expand-errors` @tjhance wrote. I imagine we don't support expanding through singular queries, maybe @tjhance can comment on whether returning `None` is correct here?

Reply:
I'd like to see this resolved, but I'd be okay with that happening after this is merged (so the branch doesn't drift).